changelogs/fragments/4377-allow-proxmox-snapshot-restoring.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - proxmox_snap - add restore snapshot option (https://github.com/ansible-collections/community.general/pull/4377).
+  - proxmox_snap - fixed timeout value to correctly relfect time in seconds (https://github.com/ansible-collections/community.general/pull/4377).
```

Comment:
```suggestion
  - proxmox_snap - fixed timeout value to correctly reflect time in seconds. The timeout was off by one second (https://github.com/ansible-collections/community.general/pull/4377).
```

Reply:
@EinDev could you please take a look at this suggestion, and the one for the tests? Thanks.